src/app/resources/resources-add.component.ts

```diff
@@ -117,7 +118,7 @@ export class ResourcesAddComponent implements OnInit {
   }
 
   onSubmit() {
-    if (this.resourceForm.valid) {
+    if (this.resourceForm.valid && !this.fileSizeExceeds) {
```

Comment:
We should set form invalid when we check file size rather than adding this other variable on the class.